### DISCUSSION TITLE
How to disable the floating menu?

### DISCUSSION THREAD

**Discussion by thhart** [GSI-CS-CO/chart-fx#401] [Q&A]
I can see a couple of plugins make usage of the menu. However even if I use some plugins I would be able to disable the menu?

**Comment by RalphSteinhagen** [comment]
@thhart this code-snippet should do the job.

```Java
  final HiddenSidesPane hiddenSidePane = chart.getPlotArea();
  hiddenSidePane.setTriggerDistance(0);
``` 

This should be all to suppress the tool bar popping up. There are other properties to change the animation, layout, rounding etc. behavioural aspects. If you want to 'pin'/keep the toolbar always open, you may use `chart.setToolBarPinned(true);`.

Hope this helps.
